src/main/java/it/gov/pagopa/cgn/portal/service/AgreementService.java

```diff
@@ -60,6 +60,11 @@ public AgreementEntity getAgreementByMerchantTaxCode(String merchantTaxCode){
 
     @Transactional
     public AgreementEntity createAgreementIfNotExists(String merchantTaxCode, EntityType entityType) {
+        return createAgreementIfNotExists(merchantTaxCode, entityType, "");
```

Comment:
We should not have this kind of method that passes a `""`.
If we want the agreement to save an organizationName we MUST make it mandatory.
All tests MUST be updated to be compliant with this.
The column on database MUST be NOT NULL with a DEFAULT value, that we will later change with the right name.